src/main/java/sx/blah/discord/handle/impl/obj/Guild.java

```diff
@@ -779,6 +778,18 @@ public IEmoji getEmojiByName(String name) {
 	}
 
 	@Override
+	public IEmoji addEmoji(String name, BufferedImage image){
```

Comment:
We want to make it as easy as possible for the user to pass in an image, while still being flexible. 

@austinv11 are you referring to `java.awt.Image` or `sx.blah.discord.util.Image`?